app/about/intro/route.js

```diff
@@ -1,16 +1,6 @@
 import Ember from 'ember'
+import Route from 'radio4000/channels/index/route'
```

Comment:
A detail, but Ember uses for its adapters/serializers the pattern `ApplicationAdapter`.

Why not continue with this as it also make obvious what this route is, `ChannelsRoute`, and not just a `Route`?

Reply:
agree, ty!